template/README.md

```diff
@@ -1,137 +1,163 @@
-# Starter template for `fredrikhgrelland/hashistack`
+<!-- markdownlint-disable MD041 -->
+<p align="center">
+ <img width="100px" src="https://www.svgrepo.com/show/58111/cube.svg" align="center" alt="Vagrant-hashistack" />
+ <h2 align="center">Vagrant-hashistack Template</h2>
+ <p align="center">Starter template for <a href="https://github.com/fredrikhgrelland/vagrant-hashistack">fredrikhgrelland/vagrant-hashistack</a></p>
+<p align="center">
+    <a href="https://github.com/fredrikhgrelland/vagrant-hashistack-template/actions">
+      <img alt="Build" src="https://github.com/fredrikhgrelland/vagrant-hashistack-template/workflows/CI/CD/badge.svg" />
+    </a>
+    <a href="https://github.com/fredrikhgrelland/vagrant-hashistack/releases">
+      <img alt="Releases" src="https://img.shields.io/github/v/release/fredrikhgrelland/vagrant-hashistack-template?label=latest%20version" />
```

Comment:
Wrong path?
<img width="656" alt="Screenshot 2020-08-25 at 16 12 15" src="https://user-images.githubusercontent.com/15572799/91185205-cf6ebc80-e6ed-11ea-8c3f-644c5e6ae342.png">

Reply:
Not the wrong path, but I'll be honest I'm not quite sure how the release process works for the template-repo. If they are directly coupled with vagrant-hashistack releases, or if they will be their own separate thing. Would be nice if there was a tag for template-releases that creates a separate release in the template repo

Reply:
Ref #270

Reply:
I'm resolving this for now, followup in #270